x-pack/test/api_integration/apis/security_solution/cases_privileges.ts

```diff
@@ -1,336 +0,0 @@
-/*
```

Comment:
These tests were moved into the cases directory here: x-pack/test/api_integration/apis/cases